### PR TITLE
Added support for XP-Pen Deco MW

### PR DESCRIPTION
**Pull request by FedericoCalzoni**
I've added support for the XP-Pen Deco MW
The SVG is just a copy of the existing XP-Pen Deco L as the layout is essentially the same.

To make this pull request I followed this other pull request [https://github.com/linuxwacom/libwacom/pull/538/commits/26707f5323eb38f81be1c00fadf03a68c65936ff](url)

### CODE REVIEW

**Comment by whot** [comment]
Thanks for the PR, but I think you mistakenly put the same `.tablet` file in as SVG?

**Comment by FedericoCalzoni** [comment]
Sorry, my bad. I tried to fix it.

**Comment by whot** [comment]
Can you add the sysinfo link to the data file please, thanks. See the [wiki](https://github.com/linuxwacom/libwacom/wiki/Tablet-Definition-Files#gathering-sysinfo)

**Comment by FedericoCalzoni** [comment]
I filed an issue with the tarball generated by the script. Let me know if I did it correctly.

File: data/xp-pen-deco-mw.tablet [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# XP-Pen
+# Deco MW
+#
```
Comment (whot) [inline data/xp-pen-deco-mw.tablet:3]:
```suggestion
#
# sysinfo.VOAFvy7j7z.tar.gz
# https://github.com/linuxwacom/wacom-hid-descriptors/issues/291
```

**Comment by whot** [comment]
merged, thanks!

**Pull request merged**